panel/theme/css/material.css

```diff
@@ -228,6 +228,10 @@ button.mdc-button.mdc-card-button {
   margin-top: 0.5rem;
```

Comment:
Wondering if this is necessary? I also noticed it making the buttons not aligned.

<img width="1339" alt="image" src="https://github.com/holoviz/panel/assets/15331990/8f5f32a8-5492-44c3-b987-0fb11bed2cc6">

Reply:
If I remove the margin-top, it's aligned
<img width="1221" alt="image" src="https://github.com/holoviz/panel/assets/15331990/f32c82b6-d92c-4d7c-8005-30d1eed51893">